src/main/java/io/ebeaninternal/server/deploy/parse/AnnotationFields.java

```diff
@@ -282,12 +297,8 @@ private void readField(DeployBeanProperty prop) {
       generatedPropFactory.setUpdateTimestamp(prop);
     }
 
-    if (get(prop, WhoCreated.class) != null) {
-      generatedPropFactory.setWhoCreated(prop);
-    }
-    if (get(prop, WhoModified.class) != null) {
-      generatedPropFactory.setWhoModified(prop);
-    }
+    initWhoCreated(prop);
+    initWhoModified(prop);
 
```

Comment:
I could merge these two methods into one (something called `initCommonAnnotationsOnFields`). That would mean one less method call with the same code quality (in my opinion).